### PR TITLE
WIP | New schedule frontend

### PR DESCRIPTION
**Pull request by pkretzschmar**

### CODE REVIEW

**Comment by pkretzschmar** [comment]
Hey guys, waiting for review and QA!

File: frontend/src/components/organisms/Frame/styles.css [hunk 1]
```diff
@@ -129,3 +129,9 @@
   z-index: 2;
   height: 100vh;
 } */
+
+@media screen and (max-width: 1000px) {
+  .drawer-paper {
+    width: 230px;
+  }
+}
```
Comment (ribeirojose) [inline frontend/src/components/organisms/Frame/styles.css:137]: 😢

File: frontend/src/components/pages/DayOff/styles.css [hunk 2]
```diff
@@ -321,10 +321,10 @@
     #show-accept { display: none; }
 }
 
-@media screen and (max-width: 720px) {
-    .drawer-paper {
+@media screen and (max-width: 1000px) {
+    /* .drawer-paper {
```
Comment (ribeirojose) [inline frontend/src/components/pages/DayOff/styles.css:325]: Could we leave this out if it is not going to be needed gain?

File: frontend/src/components/pages/Schedule/style.css [hunk 3]
```diff
@@ -2,6 +2,15 @@
...
+}
+
+
+
+@media screen and (max-width: 600px) {
+
+  #hour-table-header {
+    width: calc(100% / 3);
+  }
 }
```
Comment (ribeirojose) [inline frontend/src/components/pages/Schedule/style.css:13]: \
```suggestion
}

```

**Review by ribeirojose — APPROVED** [review]

**Comment by ribeirojose** [comment]
If this is a work in progress, you can mark it as such when opening the PR. This way, no one will be able to merge until you mark it ready. It'd also be lovely to include screenshots and/or link to access the proposed changes :D

**Pull request merged**